linalg/densemat.cpp

```diff
@@ -3503,4 +3505,117 @@ DenseTensor &DenseTensor::operator=(double c)
    return *this;
 }
 
+void BatchLUFactor(DenseTensor &Minv, Array<int> &P)
+{
+
+   int m = Minv.SizeI();
+   int NE = Minv.SizeK();
+   P.SetSize(m*NE);
+   auto data_all = mfem::Reshape(Minv.ReadWrite(), m, m, NE);
```

Comment:
`SizeI()` is `elemDofs` and `SizeK()` is `ne` in the class `EABilinearFormExtension`.